webpack/components/AnsibleRolesSwitcher/components/AvailableRolesList.js

```diff
@@ -19,7 +19,9 @@ const AvailableRolesList = ({
       <Pagination
         viewType="list"
         itemCount={itemCount}
-        pagination={pagination}
+        updateParamsByUrl={false}
+        page={pagination.page}
```

Comment:
page is also 1 by default: https://github.com/theforeman/foreman/blob/develop/webpack/assets/javascripts/react_app/components/Pagination/index.js#L146

if you don't need to manage it in this component, maybe you can also remove it from the component's props

Reply:
It doesn't work if I don't pass the pagination values, as I did here.

Reply:
Ok, I will need to debug it to see what's going on

Reply:
@liorKGOW maybe you have some time to try to debug this with Nofar?

Reply:
Sure
DM me ? @nofaralfasi